src/digitalIdentity-backend/src/test/java/didentity/amos/digitalIdentity/services/CredentialDefinitionServiceTest.java

```diff
@@ -81,10 +81,36 @@ void testCreate_withWrongParametersForCreateCredentialDefinition_shouldNotCreate
         assertEquals(httpStatusExpected, httpStatus);
     }
 
+    @Test
+    void testCreate_withCreateCredentialDefinitionReturnNull_shouldNotCreateCredentialAndReturn500() {
```

Comment:
as already adressed earlier. I do not think that you have to sperate these tests, as the ``@Autowired`` will kill the application if the lissiBackedn Service is not found